### PR TITLE
Completed html-forms exercise

### PR DESCRIPTION
**Pull request by Alex-Davis0**
Html Forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Html Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div>
+        <label for="email">Email <br /></label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: I would make sure to remove all the br tags and replace them by wrapping the associated inputs with divs. br tags are an older way of doing line breaks.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        <input type="checkbox" id="sign me up" />
+        <label for="sign me up">Sign me up!</label>
+      </div>
+      <div>
+        <label for="comments">Comments <br /></label>
+        <textarea name="comments" id="comments" cols="30" rows="10"></textarea>
+      </div>
+      <div>
+        <input type="submit" id="submit" />
+        <label for="submit"></label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: instead of a label, I would recommend the `value` attribute for the input with the value of `Submit`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Nice work on this! Just a few small updates. Let me know if you have any questions on the updates!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**